components/feature-task.html

```diff
@@ -0,0 +1,24 @@
+<div class="bg-black relative min-h-screen min-w-screen">
+    <div class="relative lg:absolute left-0 top-0 w-full lg:w-1/2 h-64 lg:h-full">
+        <img src="C:\Users\Mayank\Desktop\PYTHON PRO\mockvita\tails\README.md\Vice-City-1920x1080.png"
+            class="lg:absolute inset-0 w-full h-full object-cover">
+    </div>
+    <div class="max-w-6xl mx-auto flex items-center pb-20 pt-0 lg:pt-24 flex items-center justify-center h-full">
+        <div class="w-1/2 hidden lg:block"></div>
+        <div
+            class="relative w-full lg:w-1/2 h-full pl-12 pr-10 lg:pl-16 lg:pr-0 flex flex-col items-start justify-center">
+            <p class="text-sm uppercase font-bold text-indigo-500 tracking-wide">Cool Features</p>
+            <h2 class="text-4xl font-bold text-gray-900 mt-5 leading-tight">RAZER VICE CITY <br> FOR GAMERS BY GAMERS.
+            </h2>
+            <p class="text-gray-600 text-base mt-5">Are you ready for GAMING TOURNAMENTS!!!!!!!!!!!!!!!!!!!!!!!!!!!</p>
```

Comment:
To many exclamation marks. 🙂
```suggestion
            <p class="text-gray-600 text-base mt-5">Are you ready for GAMING TOURNAMENTS!</p>
```

Reply:
im will remove them too ! sry!